bagitobjecttransfer/recordtransfer/context_processors.py

```diff
@@ -1,15 +1,25 @@
-from recordtransfer.settings import SIGN_UP_ENABLED, MAX_SINGLE_UPLOAD_SIZE, \
-    MAX_TOTAL_UPLOAD_COUNT, MAX_TOTAL_UPLOAD_SIZE, FILE_UPLOAD_ENABLED
+from bagitobjecttransfer.settings.base import (
+    FILE_UPLOAD_ENABLED,
+    MAX_SINGLE_UPLOAD_SIZE,
+    MAX_TOTAL_UPLOAD_SIZE,
+    SIGN_UP_ENABLED,
+)
+from bagitobjecttransfer.settings.base import (
+    MAX_TOTAL_UPLOAD_COUNT,
+)
+
 
 def signup_status(request):
-    return {'SIGN_UP_ENABLED': SIGN_UP_ENABLED}
+    return {"SIGN_UP_ENABLED": SIGN_UP_ENABLED}
```

Comment:
This should be:

```suggestion
    return {"SIGN_UP_ENABLED": settings.SIGN_UP_ENABLED}
```

And the same for all the other settings in this file.

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/265/commits/c510965457dec23d9fcfbb7b45ea77955c52f5ba